lab_1/main.py

```diff
@@ -40,7 +68,17 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list or None:
     :param top_n: a number of the most common words
     :return: a list of the most common words
     """
-    pass
+    if not isinstance(freq_dict, dict) or not isinstance(top_n, int):
+        return None
+    freq_list = sorted(freq_dict.items(), key=lambda x: x[1], reverse=True)
```

Comment:
Can you sort keys just here (not items)?
 
This way you don't need to create a new list and then get top_n from it

Reply:
Done